aspnet/mvc/views/forms/sample/final/ViewModels/AddressViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace FormsTagHelper.ViewModels
+{
+    public class AddressViewModel
+    {
+        public string AddressLine1 { get; set; }
+    }
+
+    public class RegisterAddressViewModel
```

Comment:
It also makes the result a little longer and less connected - but each class in it's own file is a good rule.

![image](https://cloud.githubusercontent.com/assets/3605364/14445452/e0c210ea-0001-11e6-8f24-dd69592b486b.png)